api/src/main/java/uk/gov/dhsc/htbhf/claimant/entity/PaymentCycle.java

```diff
@@ -45,6 +45,10 @@
     @Enumerated(EnumType.STRING)
     private EligibilityStatus eligibilityStatus;
 
+    @Column(name = "qualifying_benefit_status")
+    @Enumerated(EnumType.STRING)
+    private EligibilityStatus qualifyingBenefitStatus;
```

Comment:
I'm not sure I understand why this isn't its own Enum, I thought they were going to diverge from the `eligibilityStatus`?

Reply:
It probably will diverge, but it currently has to be the same as the value that is returned by the eligibility service. I was talking about it with David on Friday, and we agreed that it will only ever have these 3 values, and that they can remain the same strings regardless of what enum we use:
```
    ELIGIBLE,
    INELIGIBLE,
    NO_MATCH,
```

Reply:
Ok, thanks for clarifying, so these values will be the translated values, not the actual values sent back to us from DWP? I was wondering if it was a record of the qualifying benefit status then it might make sense to store that value instead.

Reply:
I was thinking we'd translate 'confirmed' & 'not_confirmed' to 'ELIGIBLE' and 'INELIGIBLE'  in the eligibility service, as they have more meaning to us.

Reply:
Yes, I get that, I was just wondering if storing the exact value we get back from DWP might be useful too.